neuralpp/test/quick_tests/symbolic/interpreter_test.py

```diff
@@ -144,3 +147,25 @@ def test_sympy_interpreter():
                                                                  [SymPyVariable(x, int),
                                                                   SymPyVariable(y, int)])
     assert si.eval(min_of_three_five, dict_to_sympy_context(dict1)) == 3
+
+
+def test_sympy_interpreter_simplify():
```

Comment:
It seems we need a few more tests here, particularly involving value bindings, such as the one I mentioned `x + 2` with `x` having value `2`.